menuinst/platforms/win.py

```diff
@@ -64,6 +65,22 @@ def quick_launch_location(self) -> Path:
     def desktop_location(self) -> Path:
         return Path(windows_folder_path(self.mode, False, "desktop"))
 
+    @property
+    def terminal_profile_location(self) -> Path:
+        if self.mode == "system":
+            warnings.warn("Terminal profiles are not available for system level installs")
+            return None
+        profile_path = (
+            Path(windows_folder_path(self.mode, False, "localappdata"))
+            / "Packages"
+            / "Microsoft.WindowsTerminal_8wekyb3d8bbwe"
+            / "LocalState"
+            / "settings.json"
+        )
```

Comment:
```suggestion
        profile_path = Path(
            windows_folder_path(self.mode, False, "localappdata"),
            "Packages",
            "Microsoft.WindowsTerminal_8wekyb3d8bbwe",
            "LocalState",
            "settings.json",
        )
```

Silly thing to save a couple lines, if you like it better.